lib/Backends/Habana/Habana.cpp

```diff
@@ -126,8 +126,8 @@ class TensorHandle final {
   TensorHandle() = default;
 
   /// Constructor. Create a tensor from Glow IR Value \p V.
```

Comment:
nit: Doxygen looks stale -- NBD but maybe good to include something about `downcast64To32` since it's not trivial.